docs/user/debugging.md

```diff
@@ -71,28 +79,132 @@ More information can be found in the [View Logs with Logcat][] help article.
 
 [View Logs with Logcat]: https://developer.android.com/studio/debug/am-logcat
 
-### Command line
+#### Command line
 
 On the command line you can show all of the log output using:
 
-```
+```shell
 adb logcat
 ```
 
 This is the unfiltered output of all log messages.
 You can match for `glean` using grep:
 
-```
+```shell
 adb logcat | grep -i glean
 ```
 
 A simple way to filter for only the application that is being debugged is by using [pidcat][], a wrapper around `adb`, which adds colors and proper filtering by application ID and log level.
 Run it like this to filter for an application:
 
-```
+```shell
 pidcat [applicationId]
 ```
 
 In the above `[applicationId]` is the product's application id as defined in the manifest file and/or build script. For the Glean sample application, this is `org.mozilla.samples.glean` for a release build and `org.mozilla.samples.glean.debug` for a debug build.
 
 [pidcat]: https://github.com/JakeWharton/pidcat
+
+## Debugging the Glean SDK in iOS
+
+Unlike Android, we aren't able to export a View or similar to the `GleanDebugActivity` in Android that can process debugging commands.  Instead, iOS allows for implementation of a custom URL scheme _within the application_.  Instead of launching the application through a proxy like `GleanDebugActivity`, we can launch the application from a URL with the debugging commands embedded in the URL itself.
+
+### Available commands and query format
+
+There are 3 available commands that you can use with the Glean SDK debug tools
+
+- `logPings`: This is either true or false and will cause pings that are submitted to also be echoed to the device's log
+- `tagPings`: This command expects a string with upper and lower case letters, numbers and dashes, with a max length of 20 characters that will be used to tag the ping and cause it to appear in the Glean Debug View.
+- `sendPing`: This command expects a string name of a ping to force immediate collection and submission of.
+
+The structure of the custom URL uses the following format:
+
+`<protocol>://glean?<command 1>=<paramter 1>&<command 2>=<parameter 2> ...`
+
+Where:
+
+- `<protocol>` is the "Url Scheme" that has been added for your app (see Instrumenting the application below), such as `glean-sample-app`
```

Comment:
Do we have any recommendation for the protocol? What if the application already has a protocol there? Can we use the existing protocol? If so, can you add that as a note?

Reply:
Most (if not all) of our iOS apps already implement a custom URL scheme like this based on the application name, so it is actually preferable to use the existing one.  We are really only adding a command processor to the existing mechanism, so it makes sense to recommend using an existing one.  Other than that, I don't know what other recommendations we could make here.

Reply:
This makes sense to me! Let's call out that existing protocols can be used.

Reply:
I added this information to the beginning of the section that describes how to implement this in the app, stating that if the app already has a custom URL scheme, then you can skip to adding the `Glean.handleCustomUrl()` section.